src/order/filter/archimedean.lean

```diff
@@ -45,6 +45,45 @@ lemma tendsto_coe_rat_at_top_iff [linear_ordered_field R] [archimedean R]
 tendsto_at_top_embedding (assume a₁ a₂, rat.cast_le) $
   assume r, let ⟨n, hn⟩ := exists_nat_ge r in ⟨(n:ℚ), by assumption_mod_cast⟩
 
+lemma at_top_countable_basis_of_archimedean [linear_ordered_semiring R] [nontrivial R]
```

Comment:
I think you can remove the `[nontrivial R]` in this lemma and in `at_top_countably_generated_of_archimedean`